FlowCrypt/src/main/java/com/flowcrypt/email/api/email/model/AttachmentInfo.kt

```diff
@@ -183,5 +183,11 @@ data class AttachmentInfo(
   companion object {
     const val DEPTH_SEPARATOR = "/"
     const val INNER_ATTACHMENT_PREFIX = "inner_"
+
+    val DANGEROUS_FILE_EXTENSIONS = arrayOf(
+      "apk",
+      "appx",
+      "appxbundle"
+    )
```

Comment:
will it be safe to open .js file on Android? (will it be opened in browser or just in text editor?)

Reply:
Good question. I've tested it on an emulator and the system can't open such a file.

![image](https://github.com/FlowCrypt/flowcrypt-android/assets/2863246/b8187864-5e6e-4e0d-8c71-f7eace1181e8)

Reply:
Looks good 👍